argo_rollouts/routing-traffic-by-using-argo-rollouts.adoc

```diff
@@ -0,0 +1,35 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="routing-traffic-by-using-argo-rollouts"]
+= Routing traffic by using Argo Rollouts
+:context: routing-traffic-by-using-argo-rollouts
+
+toc::[]
+
+You can progressively route a subset of user traffic to a new application version by using Argo Rollouts and its traffic-splitting mechanisms. This allows you to test if the application is deployed and working.
+
+With Openshift Routes, you can configure Argo Rollouts to reduce or increase the amount of traffic by directing it to various applications in a cluster environment based on your requirements.
+
+You can use OpenShift Routes to split traffic between two application versions:
+
+* *Canary version*: A new version of an application where you gradually route the traffic.
+* *Stable version*: The current version of an application. After the canary version is stable and has all the user traffic directed to it, it becomes the new stable version. The previous stable version is discarded.
+
+[id="prerequisites_routing-traffic-by-using-argo-rollouts"]
+== Prerequisites
+* You have logged in to the {OCP} cluster as an administrator.
+* You have installed xref:../installing_gitops/installing-openshift-gitops.adoc#installing-openshift-gitops[{gitops-title}] on your {OCP} cluster.
+* You have installed xref:../argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc#gitops-creating-rolloutmanager-custom-resource_using-argo-rollouts-for-progressive-deployment-delivery[Argo Rollouts] on your {OCP} cluster.
+* You have installed the xref:../installing_gitops/installing-argocd-gitops-cli.adoc#installing-argocd-gitops-cli[{gitops-title} CLI] on your system.
+* You have installed the xref:../argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc#gitops-installing-argo-rollouts-cli-on-linux_using-argo-rollouts-for-progressive-deployment-delivery[Argo Rollouts CLI] on your system. 
+
+// Configuring Argo Rollouts to route traffic by using OpenShift Routes
+include::modules/gitops-configure-rollout-route-traffic-using-openshift-routes.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_routing-traffic-by-using-argo-rollouts"]
```

Comment:
```suggestion
[id="additional-resources_routing-traffic-by-using-argo-rollouts_{context}"]
```
Needs context tag, see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Reply:
Food catch! thanks, I addressed it wherever applicable.